docs/products/kafka/kafka-connect/index.rst

```diff
@@ -0,0 +1,43 @@
+Aiven for Apache Kafka Connect
+==============================
+
+What is Aiven for Apache Kafka Connect?
+---------------------------------------
+
+Aiven for Apache Kafka Connect is a fully managed **distributed Apache Kafka integration component**, deployable in the cloud of your choice. Apache Kafka Connect lets you integrate your existing data sources and sink with Apache Kafka.
+
+With a Apache Kafka Connect connector, you can source data from an existing technology into a topic or sink data from a topic to a target technology by just defining the endpoints.
+
+
+Why Apache Kafka Connect?
+-------------------------
+
+Apache Kafka represents the best in class data streaming solution. Apache Kafka Connect enables to integrate Apache Kafka with the rest of your data architecture by setting a configuration file defining the source and the target of your data.
```

Comment:
To make it easier to read I'd replace "by setting a configuration file defining the source and the target of your data." with "with a help of configuration file which defines the source and the target of your data"

Reply:
fixed